plugins/chrome-ux-report/README.md

```diff
@@ -0,0 +1,96 @@
+## What is Chrome UX Report?
+
+The Chrome User Experience Report is powered by real user measurement of key user experience metrics across the public web, aggregated from users who have opted-in to syncing their browsing history, have not set up a Sync passphrase, and have usage statistic reporting enabled. If you are curious of what is the metrics, you can go to [UX Metrics](https://developers.google.com/web/tools/chrome-user-experience-report).
+
+## Configuring the Plugin
+
+You can add this plugin to the project follow these steps.
+
+1. Run:
+
+```sh
+$ yarn add @backstage/plugin-chrome-ux-report @backstage/plugin-chrome-ux-report-backend
```

Comment:
well not really like that in the root - you have to add one to the app, and one to the backend